app/controllers/katello/api/v2/host_packages_controller.rb

```diff
@@ -4,7 +4,8 @@ class Api::V2::HostPackagesController < Api::V2::ApiController
 
     before_filter :require_packages_or_groups, :only => [:install, :remove]
     before_filter :require_packages_only, :only => [:upgrade]
-    before_filter :find_host
+    before_filter :find_host_with_facet, :except => :index
```

Comment:
Looks like this requires find_host to have been called, but find_host is restricted to only index.